### ISSUE TITLE
Version 1.5.8 / Webpack errors

### ISSUE THREAD

**Issue by bbimber** [GMOD/jbrowse-components#2644]
Hello,

We embed react-linear-genome-view in our npm project. When upgrading to 1.5.5 or 1.5.8, we now we get two things:

A lot of errors with the path /data/jbrowse-components2/:

```

ERROR in ./node_modules/@jbrowse/core/util/types/mst.js 12:55-158
Module not found: Error: Can't resolve '/data/jbrowse-components2/node_modules/@babel/runtime/helpers/esm/objectWithoutProperties.js' in 'C:\Projects\discvr-lk\server\modules\DiscvrLabKeyModules\jbrowse\node_modules\@jbrowse\core\util\types'
 @ ./node_modules/@jbrowse/plugin-linear-genome-view/dist/plugin-linear-genome-view.esm.js 24:0-65 1668:10-16 6166:8-17 6170:34-40
 @ ./src/client/JBrowse/Search/StandaloneSearch.tsx 4:0-73 118:89-108
 @ ./src/client/JBrowse/Search/webpart/app.tsx 4:0-51 6:52-68

```

and then these, which seem to be related to .js files having typescript:

```
ERROR in ./node_modules/@jbrowse/plugin-svg/src/SvgFeatureRenderer/components/Subfeatures.js 11:4
Module parse failed: Unexpected token (11:4)
You may need an appropriate loader to handle this file type, currently no loaders are configured to process this file. See https://webpack.js.org/concepts#loaders
|
|   return (
>     <>
|       {feature.get('subfeatures').map(subfeature => {
|         const subfeatureId = String(subfeature.id())
 @ ./node_modules/@jbrowse/plugin-svg/src/SvgFeatureRenderer/components/util.ts 6:0-40 18:13-24
 @ ./src/client/JBrowse/Browser/plugins/ExtendedVariantPlugin/ExtendedVariantRenderer/components/Diamond.tsx 7:0-83 30:6-11 37:16-21
 @ ./src/client/JBrowse/Browser/plugins/ExtendedVariantPlugin/ExtendedVariantRenderer/components/ExtendedVariantRenderering.tsx 7:0-32 95:13-20
 @ ./src/client/JBrowse/Browser/plugins/ExtendedVariantPlugin/ExtendedVariantRenderer/index.tsx 4:0-84 4:0-84
 @ ./src/client/JBrowse/Browser/plugins/ExtendedVariantPlugin/index.ts 10:0-152 34:44-67 36:22-46 37:20-42
 @ ./src/client/JBrowse/Browser/Browser.tsx 6:0-74 15:23-44
 @ ./src/client/JBrowse/Browser/app.tsx 3:0-29 6:52-56

```

Have you seen either of these?

**Comment by cmdcolin** [comment]
The first of these issues should be fixes by #2645, we messed up the babel config in 1.5.5-1.5.8 unfortunately and will republish a new version shortly

The second one seems weird, this might be due to a yarn linked setup with @jbrowse/plugin-svg maybe?

**Comment by cmdcolin** [comment]
should be fixed now
